components/SiteCard.tsx

```diff
@@ -1,69 +1,75 @@
-import { FC, PropsWithChildren } from 'react';
-import { DeleteIcon, EditIcon } from './icons';
 import Link from 'next/link';
+import React, { FC } from 'react';
 import { useSiteData } from '~/lib/hooks';
 import { getCurrentTimeForecast } from '~/lib/utils';
-
-import SiteGraph from './graphs/SiteGraph';
+import { DeleteIcon, EditIcon } from './icons';
 
 interface SiteCardProps {
   href?: string;
+  onClick?: () => void;
   siteUUID: string;
 }
 
-const SiteCard: FC<SiteCardProps> = ({ href, siteUUID }) => {
-  const { forecastData, client_site_name, installed_capacity_kw, isLoading } =
-    useSiteData(siteUUID);
+// eslint-disable-next-line react/display-name
```

Comment:
why is this line necessary?

Reply:
Fixed by setting `displayName` on the component.